pkg/api/admin_users.go

```diff
@@ -110,3 +110,26 @@ func AdminDeleteUser(c *m.ReqContext) {
 
 	c.JsonOK("User deleted")
 }
+
+// POST /api/admin/users/:id/logout
+func (server *HTTPServer) AdminLogoutUser(c *m.ReqContext) Response {
+	userID := c.ParamsInt64(":id")
+
+	if c.UserId == userID {
+		return Error(400, "You cannot logout yourself", nil)
```

Comment:
Why not?

Reply:
Was thinking that we maybe wanted to have this in login.go where we have the regular logout today which cleanup cookies etc and redirect to login page.

Reply:
Perhaps we should redirect the user to the logout url?

Reply:
Hard to do that when you're in the API why I suggest adding it to login.go